src/panels/lovelace/components/hui-unavailable.ts

```diff
@@ -1,57 +0,0 @@
-import {
```

Comment:
We should keep this component for cards that don't use inputs. ie Weather Card

Reply:
Do we? Why would you want that overlay if you don't need to block input? Just display unavailable as the state?

Reply:
Didnt you ask for a different view on the weather when unavailable?

In discord you had an infinite loop issue.

Reply:
That was because the old weather card displayed things that it shouldn't when unavailable. You should not render the temperature unit if there is no temperature etc.

Reply:
![image](https://user-images.githubusercontent.com/18730868/78171770-f832a480-7422-11ea-820d-2d7129d749c3.png)

Reply:
If the state is unavailable, then the card should be showing anything but unavailable.

Reply:
Those grey blocks are pretty ugly.

Reply:
So are the red warning blocks. Unavailable is not a state the weather should be in and therefore should be ugly like the warning blocks

Reply:
Red warning blocks mean the entity doesn't exist, that's an error state.

Unavailable could mean that the API is temporarily down (server maintenance), and that should be handled gracefully I think.

Reply:
With the grey block, I don't even know what is unavailable, it's confusing.

Reply:
I think that the only thing to add would be the entity name

`Entity is Unavailable: Weather.darksky`

Reply:
Nothing else is available in that state, right?

Reply:
If there is a state object that's unavailable we have entity ID and the friendly_name attribute.

Reply:
{{friendly_name}} is Unavailable

Reply:
Would maybe be nice to get a friendly message?

`{{entity_id}} is unavailable at this time. Please click for more info`

Reply:
![image](https://user-images.githubusercontent.com/18730868/78180028-28cd0b00-7430-11ea-80d3-ac06e2637348.png)

Reply:
Use `computeStateName` so you can render the name?

Reply:
What if the friendly name is `home`. 

Thats not very descriptive on which entity its talking about.

Reply:
Add entity ID in parentheses ?